js/form.js

```diff
@@ -0,0 +1,53 @@
+const form = document.getElementById('contact');
+const emailForm = document.getElementById('email-address');
+const emailError = document.querySelector('.form-error');
+const formBtn = document.querySelector('.form-btn');
+const nameFormInput = document.getElementById('full-name');
+const msgFormInput = document.getElementById('message');
+
+const storageData = (object) => {
+  localStorage.setItem('localStorageObj', JSON.stringify(object));
+};
+
+function validateEmail(email) {
+  if (!email.checkValidity()) {
+    emailError.classList.remove('error-visible');
+    return false;
+  }
+  emailError.classList.add('error-visible');
+  return true;
+}
+
+formBtn.addEventListener('click', () => {
+  validateEmail(emailForm);
+});
+
+function displayData() {
+  const data = JSON.parse(localStorage.getItem('localStorageObj')) || { name: '', msg: '', email: '' };
+  nameFormInput.value = data.name;
+  msgFormInput.value = data.msg;
+  emailForm.value = data.email;
+}
+
+function formListener() {
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+
+    if (validateEmail(emailForm)) {
+      const nameValue = nameFormInput.value;
+      const msgValue = msgFormInput.value;
+      const emailValue = emailForm.value;
+      const localStorageObj = {
```

Comment:
- [ ] Well done implementing successfully storing the form data in the browser. However, this only happens when the user submits the form. For this milestone, after filling the form, and not submitting it, the entered values should be saved in the browser. Please modify your code to meet this requirement 🙏